java/java.lsp.server/vscode/package-lock.json

```diff
@@ -10,23 +10,23 @@
 			"license": "Apache 2.0",
 			"dependencies": {
 				"jsonc-parser": "3.0.0",
-				"vscode-debugadapter": "1.51.0",
-				"vscode-languageclient": "8.0.1"
+				"vscode-debugadapter": "1.42.1",
+				"vscode-languageclient": "7.0.0"
```

Comment:
Is it intended to downgrade the versions? There are downgraded versions for more modules.

Reply:
Unintentional - seems as leftovers from local experiments were committed, no change to vscode should be made here. Will clean up.

Reply:
Changes removed (= same as `master`) in f3ee411faa